articles/quickstart/webapp/aspnet-core/interactive.md

```diff
@@ -0,0 +1,160 @@
+
+## Configure Auth0 {{{ data-action=configure }}}
+
+### Create a new application
+Create a new application or select an existing application to integrate with. You can also create and manage your applications in the Dashboard at Dashboard > Applications > Applications. 
+
+If you are logged in configuring your settings in the quickstart will automatically update your settings in the dashboard.
+
+### Configure Callback URLs
+
+A callback URL is a URL in your application where Auth0 redirects the user after they have authenticated. If not set, users won't have a place to be redirected to after logging in.
+
+### Configure Logout URLs
+
+A logout URL is a URL in your application that Auth0 can return to after the user has been logged out of the authorization server. This is specified in the returnTo query parameter. If this field is not set, users will be unable to log out from the application and will get an error.
+
+## Install and Configure the SDK {{{ data-action=code data-code="Startup.cs#9:13" }}}
+
+### Install from Nuget
+
+To integrate Auth0 with ASP.NET Core you can use our SDK by installing the `Auth0.AspNetCore.Authentication` [Nuget package](https://www.nuget.org/packages/Auth0.AspNetCore.Authentication/) to your application.
+
+```bash
+Install-Package Auth0.AspNetCore.Authentication
+```
+
+### Configure the middleware
+
+To enable authentication in your ASP.NET Core application, use the middleware provided by the SDK. Go to the `ConfigureServices` method of your `Startup` class and call `services.AddAuth0WebAppAuthentication()` to register the SDK's middleware.
+
+Ensure to configure the `Domain` and `ClientId`, these are required fields to ensure the SDK knows which Auth0 tenant and application it should use.
+
+Make sure you have enabled authentication and authorization in your `Startup.Configure` method.
+
+## Login {{{ data-action=code data-code="AccountController.cs#7:20" }}}
+
+To allow users to login to your ASP.NET MVC application, add a `Login` action to your controller.
+
+Call `HttpContext.ChallengeAsync()` and pass `Auth0Constants.AuthenticationScheme` as the authentication scheme. This will invoke the OIDC authentication handler that our SDK registers internally. Be sure to also specify the corresponding `authenticationProperties`, which you can construct using the `LoginAuthenticationPropertiesBuilder`.
+
+After succesfully calling `HttpContext.ChallengeAsync()`, the user will be redirected to Auth0 and signed in to both the OIDC middleware and the cookie middleware upon being redirected back to your application. This will allow the users to be authenticated on subsequent requests.
+
+::::checkpoint
+
+:::checkpoint-default
+
+Now that you have configured Login, run your application to verify that:
+* Navigating to your `Login` action will redirect to Auth0
+* Entering your credentials will redirect you back to your application.
+
+:::
+
+:::checkpoint-success
+Yay!
+:::
```

Comment:
I know this is a copy-paste, but we can remove the `checkpoint-success` blocks, as they aren't used. I'll clean up the React one as well.